<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QWidget" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主页</string>
  </property>
  <property name="whatsThis">
   <string>三角形中单</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#HomeWindow
{
	background-color:rgb(245, 245, 245);
}
QLabel
{
	font: 14px &quot;宋体&quot;;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="CentralLayout">
     <item>
      <widget class="QWidget" name="HeadWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#HeadWidget
{
	background-color:white;
	border-bottom: 1px solid #f0f0f0;
}
#BooksTotalLabel, #BooksBorrowingLabel, #BooksExpiredLabel
{
	font-weight:bold;
}
QPushButton
{
	border: 0;
	font: Bold 28px &quot;黑体&quot;;
	color:rgb(128, 157, 203);
	text-align: left;
}
QPushButton:pressed
{
	color:rgb(170, 0, 255);
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="BooksTotalLayout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="BooksTotalLabel">
            <property name="text">
             <string>图书总数</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="CountofBooksTotalLayout">
            <item>
             <widget class="QPushButton" name="BooksTotal">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="BooksTotalClassifier">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>本</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="BooksBorrowingLayout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="BooksBorrowingLabel">
            <property name="text">
             <string>正在借的图书</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="CountofBooksBorrowingLayout">
            <item>
             <widget class="QPushButton" name="BooksBorrowing">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="BooksBorrowingClassifier">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>本</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="BooksExpiredLayout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="BooksExpiredLabel">
            <property name="text">
             <string>已过期的图书</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="CountofBooksExpiredLayout">
            <item>
             <widget class="QPushButton" name="BooksExpired">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="BooksExpiredClassifier">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>本</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="BooksAvaliableLayout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="BooksAvaliableLabel">
            <property name="styleSheet">
             <string notr="true">font-weight: bold;</string>
            </property>
            <property name="text">
             <string>可借阅的已预约图书</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="CountofBooksAvaliableLayout">
            <item>
             <widget class="QPushButton" name="BooksAvaliable">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="BooksAvaliableClassifier">
              <property name="text">
               <string>本</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ContextLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QWidget" name="NewBookWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#NewBookWidget
{
	border: 1px solid rgb(235,235,235);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="NewBookTitleWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#NewBookTitleWidget
{
	background-color: white;
}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="NewBookTitle">
               <property name="styleSheet">
                <string notr="true">#NewBookTitle
{
	font-weight: bold;
}</string>
               </property>
               <property name="text">
                <string>新书推荐</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="RefreshNewBook">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">#RefreshNewBook
{
	border: 0;
	border-image: url(:/Icon/refresh_white.png);
}
#RefreshNewBook:pressed
{
	border-image: url(:/Icon/refresh_black.png);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="NewBookContent" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">#NewBookContent
{
	background-color: white;
}</string>
            </property>
            <layout class="QVBoxLayout" name="NewBookLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QWidget" name="NewBookCover" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QHBoxLayout" name="NewBookCoverLayout">
                <item>
                 <widget class="QLabel" name="Cover">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>200</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="Title">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">font-weight:bold;</string>
               </property>
               <property name="text">
                <string>书名</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="Author">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>作者</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="Press">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>出版社</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="RecordList">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QWidget" name="BorrowingListWidget" native="true">
           <property name="styleSheet">
            <string notr="true">#BorrowingListWidget
{
	border: 1px solid rgb(235,235,235);
	background-color: white;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QWidget" name="BorrowListTitleWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#BorrowListTitleWidget
{
	background-color: white;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="BorrowListTitle">
                 <property name="styleSheet">
                  <string notr="true">font-weight:bold;</string>
                 </property>
                 <property name="text">
                  <string>正在借的图书列表</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="RefreshBorrowList">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>15</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#RefreshBorrowList
{
	border: 0;
	border-image: url(:/Icon/refresh_white.png);
}
#RefreshBorrowList:pressed
{
	border-image: url(:/Icon/refresh_black.png);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="BorrowingList">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="SubscribeListWidget" native="true">
           <property name="styleSheet">
            <string notr="true">#SubscribeListWidget
{
	border: 1px solid rgb(235, 235, 235);
	background-color: white;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QWidget" name="SubscribeListTitleWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#SubscribeListTitleWidget
{
	background-color: white;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="SubscribeListTitle">
                 <property name="styleSheet">
                  <string notr="true">font-weight: Bold;</string>
                 </property>
                 <property name="text">
                  <string>已经预约的图书列表</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="RefreshSubscribeList">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>15</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>15</width>
                   <height>15</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#RefreshSubscribeList
{
	border: 0;
	image: url(:/Icon/refresh_white.png);
}
#RefreshSubscribeList:pressed
{
	image: url(:/Icon/refresh_black.png);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="SubscribeList">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>RefreshBorrowList</sender>
   <signal>clicked()</signal>
   <receiver>HomeWindow</receiver>
   <slot>refreshBorrowList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RefreshSubscribeList</sender>
   <signal>clicked()</signal>
   <receiver>HomeWindow</receiver>
   <slot>refreshBorrowList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RefreshNewBook</sender>
   <signal>clicked()</signal>
   <receiver>HomeWindow</receiver>
   <slot>initNewBookRecommanded()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshBorrowList()</slot>
  <slot>initNewBookRecommanded()</slot>
 </slots>
</ui>
